app/src/main/java/no/usn/mob3000/ui/screens/auth/LoginScreen.kt

```diff
@@ -1,34 +1,116 @@
 import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
 import no.usn.mob3000.R
 import no.usn.mob3000.Viewport
 
+
 /**
- * This is the chessboard page, where users can free-play against an AI or
- * fellow physically near player. (Local multiplayer, in other words).
+ * This is the login screen for authorization to an account
+ * as well as an option for logging in with a guest account.
  *
  * @see ForgotPasswordScreen
  * @see ResetPasswordScreen
- * @author frigvid
- * @created 2024-09-24
+ * @author Anarox
+ * @created 2024-09-30
  */
 @Composable
-fun LoginScreen() {
+fun LoginScreen(
+    onLogin: () -> Unit,
+    onForgot: () -> Unit
+) {
+    var username by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
     Viewport { innerPadding ->
         Box(
-            modifier = Modifier.fillMaxSize()
-                               .padding(innerPadding),
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(innerPadding),
             contentAlignment = Alignment.Center
         ) {
-            Text(stringResource(R.string.auth_login_title))
+            Column(
+                modifier = Modifier.padding(16.dp),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                // Title
+                Text(text = stringResource(R.string.auth_login_title))
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                // Username input
+                TextField(
+                    value = username,
+                    onValueChange = { username = it },
+                    label = { Text("Brukernavn") },
+                    modifier = Modifier.fillMaxWidth()
+                )
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                // Password input
+                TextField(
+                    value = password,
+                    onValueChange = { password = it },
+                    label = { Text("Passord") },
+                    modifier = Modifier.fillMaxWidth()
+                )
+
+                Spacer(modifier = Modifier.height(24.dp))
+
+                // Logg in button
+                Button(
+                    onClick = { onLogin() },
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    Text("Log In")
+                }
+
+                Spacer(modifier = Modifier.height(16.dp))
+
+                // Gjestebruker knapp
```

Comment:
The back-end doesn't support guest users. It's better to just remove this. And remember to use English.